webapp/src/components/Metrics/DeltaComputationBar.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { alpha, Box, Typography } from "@mui/material";
+import { motion } from "framer-motion";
+
+type Props = {
+  value: number;
+  formattedValue: string | number;
```

Comment:
It feels error-prone to accept an unformatted number as `formattedValue`. I would prefer removing it:
```suggestion
  formattedValue: string;
```

Reply:
I would say number type has its purpose here because "deltaUtteranceCount" is a number. but, we already send defined type values(number | string) from render cell.

Reply:
Ah, interesting. I didn't think of `deltaUtteranceCount`. Maybe it would be better to force the caller to intentionally convert to string, to make sure one didn't forget the formatting and accidentally passed a number. 🤷‍♂️ I'm not sure. You decide.
```diff
-              formattedValue={value}
+              formattedValue={String(value)}
```